game/ffmpeg.cc

```diff
@@ -302,10 +315,11 @@ class FfmpegError: public std::runtime_error {
 
 void FFmpeg::decodePacket() {
 #if LIBAVCODEC_VERSION_INT >= (AV_VERSION_INT(57, 37, 0))
-	AVPacket pkt;
```

Comment:
Is this intended? Why'd you put the declaration inside the loop?

Reply:
Yes this is intendended for now, I need more tests anyway. Don't bother too much with reviews for now, the purpose of the PR was mainly to know your opinion on the legacy AV lib we want to support. I'll let you know when the code is ready for reviews.

Reply:
Setting this as 'resolved'